### PR TITLE
clean up duplicated codes

### PR DESCRIPTION
**Pull request by cwdsuzhou**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:
clean up duplicated codes

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
None
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Comment by cwdsuzhou** [comment]
ping @ravisantoshgudimetla  for review

File: pkg/scheduler/internal/queue/scheduling_queue.go [hunk 1]
```diff
@@ -370,27 +370,18 @@ func (p *PriorityQueue) flushBackoffQCompleted() {
 			return
 		}
 		pod := rawPodInfo.(*framework.PodInfo).Pod
-		boTime, found := p.podBackoff.GetBackoffTime(nsNameForPod(pod))
-		if !found {
-			klog.Errorf("Unable to find backoff value for pod %v in backoff queue", nsNameForPod(pod))
-			p.podBackoffQ.Pop()
+		if !p.isPodBackingOff(pod) {
```
Comment (ahg-g) [inline pkg/scheduler/internal/queue/scheduling_queue.go]: This is semantically different from the current code. The current code continues to move the pod even if it was not found in the podBackoff map. 

I don't know though if there are corner cases where the podBackoff queue and map are out of sync, in which case we should stick with the current logic.

Comment (cwdsuzhou) [reply pkg/scheduler/internal/queue/scheduling_queue.go]: Thanks for review, I would take a closer look at the current logic to check if inconsistency may exist.

**Pull request closed without merging**